### ISSUE TITLE
Suggestion: Import POH Tele's into Calculations

### ISSUE THREAD

**Issue by SlothDotEXE** [Skretzo/shortest-path#19]
Example -
You're in Port Sarim and you set your target to Taverley. You have a Falador Teleport portal inside of your POH.

Expected Result - 
It tells you to tele POH, then take the Falador Portal, and then run to Taverley, as there's less tiles ran.

Current Result -
It tells you to run the entire path, taking agility shortcuts if necessary.

I think this would be a good addition to the plugin as it would enable users an even 'shorter path' to their targets.

**Comment by Skretzo** [comment]
This is something I want to implement, but it will require a substantial rework of the plugin. Fortunately I have had some help on this, but the work is currently stale.

**Comment by mikeweyandt** [comment]
This also seems to happen just with teleports in general (unless I have something configured)

i.e. in Port Sarim, target Taverly. It suggests walking all the way versus in theory checking magic level (or POH) for fally tele, then doing that path.

I know this is an old issue, any updates to this or is the stale work in a particular branch?

**Comment by mikeweyandt** [comment]
Yeah, would be interesting to do what quest helper does with items in bank/on you/etc to make those calls. But gotcha, yeah tele does feel more nice to have if all the shortcuts/other things aren't present yet

**Comment by Infinitay** [comment]
Would it be possible to add extra settings to the plugin that allows a user to specify the various teleport methods they have inside their PoH and to utilize those instead? I know you planned on refactoring and making whole system to handle PoH teleports, but in the meantime what if you were to provide a simple substitute instead? So instead of checking to see if a player has a required spell/item teleport, override it to having the item in your inventory or the spell available if the user marked it as available in their PoH. I hope that makes sense.

**Comment by Skretzo** [comment]
> Would it be possible to add extra settings to the plugin that allows a user to specify the various teleport methods they have inside their PoH and to utilize those instead? I know you planned on refactoring and making whole system to handle PoH teleports, but in the meantime what if you were to provide a simple substitute instead? So instead of checking to see if a player has a required spell/item teleport, override it to having the item in your inventory or the spell available if the user marked it as available in their PoH. I hope that makes sense.

The hard part is not actually to include the teleports, but rather where the teleport portal can be. The PoH is instanced which messes up coordinates and you can place the portal chamber almost anywhere inside your house (even upstairs).

**Comment by Infinitay** [comment]
Oh right I completely ignored that fact; however, assuming you go down the route I suggested in which PoH will just be a substitute for items/spells, you could just think of it with a weight of one - as if the item or spell is in your inventory. Ignoring the distance of the portals just so we can at the least have basic PoH support. Or if you want, you could assign it a default weight of say 5 or 6 if you either still want it to have a weight or if you want to separate it from actual items/spells.
